spec/linter-stylelint-spec.js

```diff
@@ -213,6 +217,34 @@ describe('The stylelint provider for Linter', () => {
     });
   });
 
+  describe('works with HTML files and', () => {
+    it('works with stylelint-config-standard', () => {
+      atom.config.set('linter-stylelint.enableHtmlLinting', true);
```

Comment:
It's get reset on an onBefore. I placed it in the describe and it took no effect being set to true.